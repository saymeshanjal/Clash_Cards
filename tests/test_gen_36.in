100
14 79 4
16 122 5
10 119 3
14 182 1
14 31 2
4 86 3
4 196 1
12 11 1
12 157 5
6 186 4
16 160 5
16 103 4
10 36 5
10 178 4
6 136 1
14 137 4
2 155 2
12 176 1
10 125 3
2 175 1
2 167 1
2 158 4
4 51 2
6 55 2
2 65 4
14 63 3
6 175 3
8 12 1
16 169 3
2 137 1
8 106 5
14 106 3
2 11 4
16 141 5
16 30 2
2 60 1
8 71 3
2 10 1
14 89 2
2 17 4
6 132 2
8 90 5
8 98 2
12 193 3
16 57 3
8 73 1
14 84 4
2 93 4
16 179 3
16 67 2
4 86 4
10 153 1
6 180 4
8 66 5
4 17 2
14 100 2
16 130 2
16 83 2
4 122 1
10 170 3
2 11 1
2 103 5
6 105 5
14 180 3
10 118 1
8 52 3
10 91 4
4 58 4
8 163 3
2 47 3
2 79 4
16 191 3
16 73 5
8 127 3
6 196 3
6 87 5
14 75 1
12 196 4
16 191 2
10 143 1
6 120 5
10 7 3
12 110 3
16 57 5
14 19 3
8 13 1
10 188 3
2 177 5
14 17 1
16 29 2
16 135 2
14 195 1
10 64 1
8 0 5
6 182 2
10 82 5
14 102 5
10 183 5
14 181 4
16 13 3
