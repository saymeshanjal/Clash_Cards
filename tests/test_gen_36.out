910 1520 1002
4248 8622 0
110 142 0
1716 1716 0
81 308 3043
1 5 0
3 3 0
410 514 0
219 705 0
1 14 5
4248 8622 0
3809 5406 3655
44 208 0
44 133 75
10 10 0
910 1520 1002
0 1 1
462 462 0
110 142 0
1 1 0
1 1 0
0 2 0
1 2 3
5 6 9
0 2 0
1652 1780 0
6 14 0
31 39 0
6307 6563 0
1 1 0
8 62 0
1652 1780 0
0 2 0
4248 8622 0
233 1179 11458
1 1 0
27 43 0
1 1 0
1078 1079 1275
0 2 0
5 6 9
8 62 0
20 21 29
430 494 0
4020 8850 0
35 35 0
910 1520 1002
0 2 0
6307 6563 0
3362 3597 5911
0 5 1
126 126 0
1 14 5
8 62 0
1 2 3
1078 1079 1275
4081 4082 4707
4081 4082 4707
3 3 0
110 142 0
1 1 0
0 2 0
1 19 0
1652 1780 0
126 126 0
27 43 0
44 133 75
0 5 1
27 43 0
0 2 0
0 2 0
6307 6563 0
2679 10191 0
27 43 0
6 14 0
1 19 0
1716 1716 0
208 441 275
4081 4082 4707
126 126 0
1 19 0
27 225 0
430 494 0
1023 11847 0
391 3041 0
33 37 0
110 142 0
0 2 0
1582 1850 0
269 1241 11360
4081 4082 4707
1716 1716 0
126 126 0
0 70 0
5 6 9
45 207 0
987 2445 0
45 207 0
910 1520 1002
1622 11248 0
